src/components/calltree/ProfileCallTreeContextMenu.js

```diff
@@ -90,6 +118,11 @@ class ProfileCallTreeContextMenu extends PureComponent {
           >
             Function Name
           </MenuItem>
+          {this.isJS()
```

Comment:
Do you mean like
```javascript
render() {
  const isJS = this.isJS()
  return ...
}
```
or simply having the body of the `isJS` function at the top of `render` and assigning the local variable right there?